tests/integration/dbclient/appl/src/test/java/io/helidon/tests/integration/dbclient/appl/it/mapping/MapperIT.java

```diff
@@ -43,31 +41,31 @@
  */
 public class MapperIT {
 
-    private static final Logger LOGGER = Logger.getLogger(MapperIT.class.getName());
+    private static final System.Logger LOGGER = System.getLogger(MapperIT.class.getName());
 
     private final TestServiceClient testClient = TestClient.builder()
             .port(HelidonProcessRunner.HTTP_PORT)
             .service("Mapper")
             .build();
 
     private void executeInsertTest(final String testName, final int id) {
```

Comment:
We use method ordering as well - first `public`, last `private`...

Reply:
moved to the end of the file